scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -1205,7 +1243,7 @@ void BPF_STRUCT_OPS(layered_dispatch, s32 cpu, struct task_struct *prev)
 
 				if (bpf_cpumask_test_cpu(cpu, layer_cpumask) ||
 				    (cpu <= nr_possible_cpus && cpu == fallback_cpu &&
-				     layer->nr_cpus == 0)) {
+				     MEMBER_VPTR(layer, ->nr_cpus) == 0)) {
```

Comment:
@JakeHillion 😄

Reply:
Why is there no angery react on GitHub 😡